packages/11ty/_plugins/shortcodes/figure.js

```diff
@@ -34,7 +34,11 @@ module.exports = function (eleventyConfig, { page }) {
     /**
      * Merge figures.yaml data and additional params
      */
-    let figure = id ? getFigure(id) : {}
+    let figure = getFigure(id)
+    if (!figure) {
+      console.warn(`Warning: The figure id "${id}" was found in the template "${page.inputPath}", but is not defined in "figures.yaml"`)
```

Comment:
We might start making our console message fancy to help them stand out to user, perhaps something like
```suggestion
      const message = chalk.yellow(`Warning: The figure id "${id}" was found in the template "${page.inputPath}", but is not defined in "figures.yaml"`)
      console.warn(message)
```

Reply:
Let's make the chalk drawing part of a separate feature pull-request...